index.bs

```diff
@@ -651,3 +779,81 @@ The [=remote end steps=] are:
 
 2. Return [=success=] with data |body|
 
+### Subscribe ### {#command-session-subscribe}
+
+The <dfn export for=commands>subscribe</dfn> command enables certain events either
+globally or for a set of browsing contexts
+
+Issue: This needs to be generalised to work with realms too
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      ```
+      SubscribeCommand = {
+        method: "session.subscribe",
+        params: SubscribeParameters
+      }
+
+      SubscribeParameters = {
+        events: [text],
+        contexts?: [BrowsingContext],
+        * text => any,
+      }
+      ```
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+      ```
+        EmptyResult
+      ```
+   </dd>
+</dl>
+
+The [=remote end steps=] with |command parameters| are:
+
+    1. Let the |list of event names| be the value of the <code>events</code> field of
+       |command parameters|.
+
+    1. Let the |list of contexts| be the value of the <code>contexts</code>
+       field of |command parameters| if it is present or null if it isn't.
+
+    1. Return the result of [=updating the event map=] with |list of event
+       names|, |list of contexts| and the target status set to true.
```

Comment:
```suggestion
       names|, |list of contexts| and |target status| true.
```

An argument isn't quite "set to" something, it's just given.

(Modulo suggested renaming elsewhere.)